"Fleet Desktop menu bar icon" and "Fleet Desktop app" (#47858)
fleetdm/fleet: docs/Get started/anatomy.md

- **Fleetd Chrome extension:** enrolls ChromeOS devices in Fleet. [Docs](https://github.com/fleetdm/fleet/blob/main/ee/fleetd-chrome/README.md).

## Fleet Desktop
Fleet Desktop is a menu bar icon that's automatically deployed as part of fleetd. Fleet Desktop gives end users visibility into the security and status of their machine. [Docs](https://fleetdm.com/docs/using-fleet/fleet-desktop).

> For macOS, you can also deploy a separate Fleet Desktop app. [Learn more](https://fleetdm.com/guides/fleet-desktop). 

Author: ```suggestion
For macOS, you can optionally deploy the Fleet Desktop app. [Learn more](https://fleetdm.com/guides/fleet-desktop). 
```
